screens/pilot/PilotProfileImageUploadScreen.js

```diff
@@ -129,13 +131,16 @@ function PilotProfileImageUploadScreen(
         <Text style={styles.completeButtonText}>Complete Profile</Text>
       </TouchableOpacity>
 
-      <TouchableOpacity
-        style={styles.backButton}
-        onPress={() => props.navigation.goBack()}
-        title={"Back"}
-      >
-        <Text style={styles.backButtonText}>Back</Text>
-      </TouchableOpacity>
+      <View style={styles.backButtonWrapper}>
+        <TouchableOpacity
+          style={styles.backButton}
+          onPress={() => props.navigation.goBack()}
```

Comment:
Instead of calling `() => props.navigation.goBack()` in the `onPress` of the component, like this:

```javascript
return (
  // component
  <TouchableOpacity
    style={styles.backButton}
    onPress={() => props.navigation.goBack()}
  // component
```
It's cleaner to destructure `goBack` before the JSX is returned, and then only pass in the function itself, like this:

```javascript
const { goBack } = props.navigation;

return (
  // component
  <TouchableOpacity
        style={styles.backButton}
        onPress={goBack}
   >
  // component
```

Reply:
Yes, that is sooooOOO much better. Will most def start doing this!!!